### PR TITLE
classes article created

### PR DESCRIPTION
**Pull request by pablonnaoji**
Seeking feedback on my Python classes article.

### CODE REVIEW

**Comment by pablonnaoji** [comment]
I went ahead and signed the contributor license agreement. Do let me know if you need anything else.

File: content/python/encyclopedia/classes-in-python.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+---
+-->
+
+# What are Classes in Python?
+
+Classes are used to define the behavior, attributes and functions of objects in code. We use the `class` keyword to define a class and use properties and methods to determine the structure and functionality of the class. In doing so a class becomes a protoype or template for instances of the object. Lets demonstrate a class for an Employee:
+
+## An example of an "Employee" Class
+
+```employee.py
```
Comment (sonnynomnom) [inline content/python/encyclopedia/classes-in-python.md:18]:
```
```py
```

File: content/python/encyclopedia/classes-in-python.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
```
Comment (sonnynomnom) [inline content/python/encyclopedia/classes-in-python.md:19]: extra line here (delete)

File: content/python/encyclopedia/classes-in-python.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    def working(self,employee_name):
+       self.name = employee_name
+       print(employee_name + " is working")
+```
+
+Once we create an `Employee` class like we did above we can import the module in our code and create instances of it whenever and wherever we want, thereby enabling code reusablity and making our code more efficient and easier to maintain.
+
+```work.py
+from employee import Employee
+
```
Comment (sonnynomnom) [inline content/python/encyclopedia/classes-in-python.md:37]: two linebreaks here (just do one)

File: content/python/encyclopedia/classes-in-python.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+# What are Classes in Python?
+
+Classes are used to define the behavior, attributes and functions of objects in code. We use the `class` keyword to define a class and use properties and methods to determine the structure and functionality of the class. In doing so a class becomes a protoype or template for instances of the object. Lets demonstrate a class for an Employee:
+
+## An example of an "Employee" Class
+
+```employee.py
+
+class Employee(object):
```
Comment (sonnynomnom) [inline content/python/encyclopedia/classes-in-python.md:20]:
```py
class Employee(object):
    name = "Sam"
    company = "ILoveCode Inc."
    age = 30
    is_on_vacation = True

    def working(self,employee_name):
       self.name = employee_name
       print(employee_name + " is working")
```

File: content/python/encyclopedia/classes-in-python.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+Subjects:
+  - "What are Classes in Python?"
+Tags: 
+  - "python"
+  - "classes"
+  - "object oriented"
+---
+-->
+
+# What are Classes in Python?
```
Comment (sonnynomnom) [inline content/python/encyclopedia/classes-in-python.md:12]:
```
# Classes in Python
```

File: content/python/encyclopedia/classes-in-python.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+Classes are used to define the behavior, attributes and functions of objects in code. We use the `class` keyword to define a class and use properties and methods to determine the structure and functionality of the class. In doing so a class becomes a protoype or template for instances of the object. Lets demonstrate a class for an Employee:
+
+## An example of an "Employee" Class
```
Comment (sonnynomnom) [inline content/python/encyclopedia/classes-in-python.md:16]:
```
## Example
```

File: content/python/encyclopedia/classes-in-python.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+class Employee(object):
+
+    name = "Sam"
+    company = "ILoveCode Inc."
+    age = 30
+    is_on_vacation = True
+
+
+    def working(self,employee_name):
```
Comment (sonnynomnom) [inline content/python/encyclopedia/classes-in-python.md:28]: add a space after comma:

```
def working(self, employee_name):
```

File: content/python/encyclopedia/classes-in-python.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+```work.py
+from employee import Employee
+
+
+def create_employee():
+   print("emplyee is starting thier job")
+   employee1 = Employee(name="Blake")
+   name = employee1.name
+   print(employee1.work(name))
+   >>> Blake is working
```
Comment (sonnynomnom) [inline content/python/encyclopedia/classes-in-python.md:44]: we shouldn't do triple >>> here. maybe just:

```
# Output: Blake is working
```

**Review by sonnynomnom — COMMENTED** [review]
left some comments! fix and we gucci!

File: content/python/encyclopedia/classes-in-python.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
+<!--
+---
+Subjects:
+  - "What are Classes in Python?"
```
Comment (sammillersmith) [inline content/python/encyclopedia/classes-in-python.md:4]: The "Subjects" field is being used as an additional tagging system. Please pick tags from Subjects that are already listed on Author here: https://author.codecademy.com/browse/categories.

File: content/python/encyclopedia/classes-in-python.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+Tags:
```
Comment (sammillersmith) [inline content/python/encyclopedia/classes-in-python.md:5]: Please use tags listed here: https://author.codecademy.com/browse/tags
"python" is on that list, but the other two are not

File: content/python/encyclopedia/classes-in-python.md [hunk 11]
```diff
@@ -0,0 +1,47 @@
+<!--
+---
```
Comment (sammillersmith) [inline content/python/encyclopedia/classes-in-python.md:2]: Please add the Title field, [as in the template](https://github.com/Codecademy/codepedia-content/blob/main/documentation/encyclopedia-entry-template.md)

File: content/python/encyclopedia/classes-in-python.md [hunk 12]
```diff
@@ -0,0 +1,47 @@
...
+Subjects:
+  - "What are Classes in Python?"
+Tags: 
+  - "python"
+  - "classes"
+  - "object oriented"
+---
```
Comment (sammillersmith) [inline content/python/encyclopedia/classes-in-python.md:9]: Please add the Catalog Content field, [as in the template](https://github.com/Codecademy/codepedia-content/blob/main/documentation/encyclopedia-entry-template.md)

**Review by sammillersmith — CHANGES_REQUESTED** [review]
Good start! Please take a look at our edits and resubmit, and you should be good to go!
Quick reminder to use the template and content standards in the [documentation folder](https://github.com/Codecademy/codepedia-content/tree/main/documentation)--those should answer any questions you might have about the metadata

**Comment by pablonnaoji** [comment]
Thanks for the feedback! I've made the requested changes. Resubmitting now.

**Pull request closed without merging**